plonetheme/barceloneta/theme/less/barceloneta-compiled.css

```diff
@@ -3922,6 +3942,25 @@ ul.picker__list {
 #content {
   margin-bottom: 48px;
 }
+#content-core a {
+  border-bottom: 1px solid #cccccc;
+}
+#content-core a:hover,
+#content-core a:focus {
+  border-bottom: none;
+}
+#content-core a:focus {
+  outline: thin dotted;
+  outline: 5px auto -webkit-focus-ring-color;
+  outline-offset: -2px;
+}
+#content-core .pat-structure a,
+#content-core .pat-resourceregistry a {
+  border-bottom: none;
```

Comment:
not quite getting your first point; there is no underline in current master on foldercontents either, as far as I can see.

on second one: yeah. We have some more wiggle room, since contrast needs to be only 3.0 as it's large text. (and I should have caught it earlier, it was hidden because of a slew of false positives). But still, we have to go darker. 

I think the best i can do would be #7099bf, which is

![selection_003](https://cloud.githubusercontent.com/assets/456196/10716464/4fecaeb4-7b3a-11e5-8408-c85cad321f33.png)

and is far less disruptive. It *just* hits the 3.0 contrast, so would do.